lib/northbound.c

```diff
@@ -773,8 +773,18 @@ static void nb_update_candidate_changes(struct nb_config *candidate,
 		root = yang_dnode_get(running_config->dnode, xpath);
 		/* code */
 		break;
-	default:
+	case NB_OP_MOVE:
+	case NB_OP_PRE_VALIDATE:
+	case NB_OP_APPLY_FINISH:
+	case NB_OP_GET_ELEM:
+	case NB_OP_GET_NEXT:
+	case NB_OP_GET_KEYS:
+	case NB_OP_LOOKUP_ENTRY:
+	case NB_OP_RPC:
 		break;
+	default:
```

Comment:
I think default is not allowed in FRR coding conventions. Last time I got this comment from Donald.